lists.go

```diff
@@ -42,6 +42,31 @@ func (a TwitterApi) AddMultipleUsersToList(screenNames []string, listID int64, v
 	return list, r.err
 }
 
+// RemoveUserFromList implements /lists/members/destroy.json
+func (a TwitterApi) RemoveUserFromList(screenName string, listID int64, v url.Values) (users []User, err error) {
+	v = cleanValues(v)
+	v.Set("list_id", strconv.FormatInt(listID, 10))
+	v.Set("screen_name", screenName)
+
+	var addUserToListResponse AddUserToListResponse
```

Comment:
Is this actually the correct response?

Reply:
Thank you for your review. It's my mistake.:disappointed_relieved:
`/lists/members/destroy.json` returns the list.
Fixed the return type of `RemoveUserFromList`.